src/test/addrman_tests.cpp

```diff
@@ -296,23 +296,23 @@ BOOST_AUTO_TEST_CASE(addrman_find)
     addrman.Add(addr2, source2);
     addrman.Add(addr3, source1);
 
-    // Test 17: ensure Find returns an IP matching what we searched on.
+    // Test: ensure Find returns an IP matching what we searched on.
     CAddrInfo* info1 = addrman.Find(addr1);
     BOOST_CHECK(info1);
     if (info1)
```

Comment:
I was going to suggest you update this if statement to follow our style guidelines (continuation if the block is one line, or braces), but really I don't think an if statement has any business being here at all. This is a test case - we should be asserting all our expectations (and indeed we do in the line above).

Suggest you remove the if line and run `BOOST_CHECK(info1->ToString() == "250.1.2.1:8333")` directly. Same for other 2 ifs below.